server/tests/model.specs/articles.specs.js

```diff
@@ -1,30 +1,45 @@
 import mocha from "mocha";
 import chai from "chai";
+import chaiAsPromise from "chai-as-promised";
 import database from "../../models/index";
-import seeds from "./mode.fixtures";
+import seeds from "./model.fixtures";
```

Comment:
seeds don't from fixtures. It should be` import {Some form of mockData} from "./fixtures/fixturefolder";`